eurovision/notebooks/story.ipynb

```diff
@@ -832,8 +832,18 @@
     "(temp.groupby('to_country')['total_points']\n",
     " .agg(['count', 'mean'])\n",
     " .query('count >= 5')\n",
+    " .round({'mean': 2})\n",
     " .sort_values(by='mean', ascending=False)\n",
-    " .head())"
+    " .head()\n",
+    " )"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "While summary statistics like the mean can be informative, it would be really interesting to see how well a country is doing over the years in a bit more detail. Are the top performing countries always much better than average, or do a few great perfomances lift up the average scores of these countries? To visualise for different countries, we're creating a function to plot the scores of a country over time and mark specific years where the country either didn't perform in the Final at all or where it has actually won the competition."
```

Comment:
This may be just me, but because "mean" also means "mean", I stumbled on this sentence. It might be easier to read with "average" instead of "mean", if that doesn't take anything away?

Reply:
Thanks, changed!